flags.html

```diff
@@ -5,79 +5,79 @@
     </head>
     <body>
 <div id="indonesia" class="flagContainer">
-
+    <div class="indonesiaRed"></div>
 </div>
-<div id="columbia" class="flagContainer">
+<!--<div id="columbia" class="flagContainer">-->
```

Comment:
why comment out everything else?

Reply:
I wanted to have just one div light up at a time on my IDE.  
<img width="593" alt="Screen Shot 2019-07-06 at 6 13 11 PM" src="https://user-images.githubusercontent.com/47374317/60762629-c7e7e200-a019-11e9-930d-afe3a713bdbc.png">